streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -4809,11 +4768,14 @@ public void shouldNotFailForTimeoutExceptionOnCommitWithEosAlpha() {
             exception.corruptedTasks(),
             equalTo(Collections.singleton(taskId00))
         );
+
+        Mockito.verify(mockitoConsumer, times(2)).groupMetadata();
```

Comment:
Mockito claims the method is called twice. This wasn't specified in EasyMock world, but I decided to make it explicit now.

Reply:
should it be called twice as per code?

Reply:
I verified that if I change
```
expect(consumer.groupMetadata()).andStubReturn(null);
```
to
```
expect(consumer.groupMetadata()).andReturn(null).times(2);
```
the test still passes. I suspect it was always called twice, just never verified